mica-core/src/main/java/org/obiba/mica/core/repository/CommentsRepository.java

```diff
@@ -19,4 +19,5 @@
   List<Comment> findByClassName(String name);
   List<Comment> findByInstanceId(String id);
   List<Comment> findByResourceIdAndInstanceId(String name, String id);
```

Comment:
This method would all comments (including the private ones). I'd suggest using the `@Query` annotation to avoid having to change the method name everywhere.

An example can be found in "mica-core/src/main/java/org/obiba/mica/core/repository/PersonRepository.java". That way you can keep the name and add more conditions.